test/helpers.tsx

```diff
@@ -27,7 +27,9 @@ export const getElements = (
 };
 
 export const getDummyHvProps = () => ({
-  onUpdate: () => {},
+  onUpdate: () => {
+    /** PASS */
```

Comment:
curious why this is needed

Reply:
lint fails, complaining that unexpected arrow function.